.github/workflows/ci.yml

```diff
@@ -60,6 +60,7 @@ jobs:
           echo "run-on-tokio-explicit: ${{ needs.run-on-tokio-explicit.result }}"
           echo "run-on-tokio-implicit: ${{ needs.run-on-tokio-implicit.result }}"
           echo "run-on-perf-event-open-sys2: ${{ needs.run-on-perf-event-open-sys2.result }}"
+          echo "fails-on-1-64-rustc-rust-fnv: ${{ needs.fails-on-1-64-rustc-rust-fnv.result }}"
```

Comment:
You'll need to add this to the `needs:` list above, that's why this job is failing right now.

After that this is ready to merge.